qiskit/backends/ibmq/ibmqjob.py

```diff
@@ -47,15 +47,15 @@ def __init__(self, qobj, api, is_device):
         """IBMQJob init function.
 
         Args:
-            qobj (dict): job description
+            qobj (QObj): job description
             api (IBMQuantumExperience): IBM Q API
             is_device (bool): whether backend is a real device  # TODO: remove this after Qobj
```

Comment:
Do we still need this TODO?

Reply:
I think so, yes - as the `Qobj` for this PR does not contain the real schema.